src/phoebus_utils/reduction.hpp

```diff
@@ -32,10 +32,17 @@ Real inline Min(const Real &x) {
   return xmin;
 }
 
+Real inline Sum(const Real &x) {
```

Comment:
Not actually used now but I think we can leave this in in case the need arises in a problem generator file at some point